lib/postcode_validation/use_case/format_validator.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'format_validators/nl_postcode_validator'
+require_relative 'format_validators/be_postcode_validator'
+require_relative 'format_validators/gb_postcode_validator'
+require_relative 'format_validators/fr_postcode_validator'
+
+class FormatValidator
+   def for country
+     case country
+     when 'NL'
+       NLPostcodeValidator.new
```

Comment:
> Are we thinking they will evolve to contain more functionality over time?

Yes, potentially.

Sure GB might be simple, but what if Japan requires some turing complete check?